### PR TITLE
Feature add limesurvey

### PR DESCRIPTION
**Pull request by LiquidITGuy**
few words about this MR : 
- there is no official docker version of Limesurvey but I take the most popular (more than 1M download)
- this docker version has a mysql database inside but is not settable by env var and retrieve informations is hard for the user (it need to retrieve log). As recommended on the documentation I made a second database to connect Limesurvey

- [X] I have tested the template using the method described in README.md thoroughly
- [X] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [X] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [X] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [X] Icon is added as a png file to the logos directory.
- [X] Documentation field contains a link to a page with proper explanations on environmental variables, volumes or the docker compose file.

### CODE REVIEW

**Comment by LiquidITGuy** [comment]
Capture d’écran 2020-05-23 à 22 35 29
Capture d’écran 2020-05-23 à 22 36 26
Capture d’écran 2020-05-23 à 22 36 30

File: public/v2/apps/limesurvey.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    },{
+      "id": "$$cap_db_name",
+      "label": "database name",
+      "defaultValue": "limesurvey",
+      "description": "name for the database."
+    },{
+      "id": "$$cap_db_user",
+      "label": "database user",
+      "defaultValue": "limesurvey",
+      "description": "Username for the database using mysql."
```
Comment (githubsaturn) [inline public/v2/apps/limesurvey.json:66]: valid regex is missing from multiple params. Please add them.

Comment (LiquidITGuy) [reply public/v2/apps/limesurvey.json]: done

**Comment by LiquidITGuy** [comment]
not tried again after dev fixes (only regex changes)

**Pull request merged**